Jenkinsfile

```diff
@@ -84,3 +83,32 @@ demos['databound'] = {
 }
 
 parallel(demos)
+
+def branchName = currentBuild.projectName
+if (!branchName.startsWith('PR-')) {
```

Comment:
Maybe it makes sense tostart from a master branch and "PR-99" so that you can actually test deployment in this PR.

Reply:
:+1:

Reply:
@fcojfernandez I do not think it has been resolved so far.
Also, I think it still makes sense to build an image for each PR as a part of the flow. We just should not push it

Reply:
Right! It is not committed yet

Reply:
Done! I just wanted to commit everything in a single commmit